app/models/user.rb

```diff
@@ -23,7 +23,7 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
-  has_many :jobs
+  has_many :jobs, through: :companies
```

Comment:
@rohan-techfreak Thanks for your contribution. If possible can you move this code to be after the companies association? That way it reads more easily and naturally follows.